src/configuration/services/redis.md

```diff
@@ -31,6 +57,8 @@ The format exposed in the ``$PLATFORM_RELATIONSHIPS`` [environment variable](/de
 }
 ```
 
+The format is identical regardless of whether it's a Persistent or Ephemeral service.
```

Comment:
In this sentence, persistent and ephemeral aren't proper nouns, so I think they should be lowercase.